packages/manager/modules/pci/src/projects/projects.service.js

```diff
@@ -63,4 +64,13 @@ export default class {
       })
       .$promise.then((quotas) => map(quotas, (quota) => new Quota(quota)));
   }
+
+  getOrderFollowUp(orderId) {
+    const endPoint = `/me/order/${orderId}/followUp`;
+    return this.$http.get(endPoint).then(({ data }) => data);
+  }
+
+  getApiSchemas() {
+    return this.$http.get('/me.json').then(({ data }) => data);
```

Comment:
```suggestion
  getMeSchema() {
    return this.$http.get('/me.json').then(({ data }) => data);
```
as only one schema is fetched

Reply:
Yes only one.
Don't need others.